src/components/AppHeader.vue

```diff
@@ -1,50 +1,51 @@
 <template>
   <div class="header">
     <div class="header-upper">
-      <h1>Shopping List App</h1>
+      <h1>{{ $t("appheader") }}</h1>
       <div class="expand-form-btn">
         <button
           v-if="showForm == false"
           class="custom-btn btn-edit-items"
           @click="toggleForm"
         >
-          <span>Expand Form</span><span>Edit List !</span>
+          <span>{{ $t("buttons.showform.span1") }}</span
+          ><span>{{ $t("buttons.showform.span2") }}</span>
         </button>
         <button
           v-if="showForm == true"
           class="custom-btn btn-edit-cancel"
           @click="toggleForm"
         >
-          <span> Close Edit</span>
+          <span> {{ $t("buttons.closeform") }}</span>
         </button>
       </div>
 
       <div class="show-modal-btn">
-        <AppButton
+        <app-button
           id="show-modal"
           class="app-p-sm"
           variant="add-item"
           @click="$emit('open')"
         >
-          Add item</AppButton
+          {{ $t("buttons.showmodal") }}</app-button
         >
       </div>
     </div>
 
     <div class="header-lower">
       <Transition name="slide-fade">
         <div v-if="showForm" class="expanded-input-space" @load="timeOutFunc">
-          <InputField @new-item="sendNewItemAgain"> </InputField>
+          <input-field @new-item="sendNewItemAgain"> </input-field>
         </div>
       </Transition>
 
       <div class="app-list-area">
-        <AppList
+        <app-list
```

Comment:
Your components should always follow PascalCase. kebab-case is for what you have not coded.